chpl/chpl-service/src/main/java/gov/healthit/chpl/dao/impl/DeveloperDAOImpl.java

```diff
@@ -335,6 +347,15 @@ private void update(DeveloperEntity entity) {
 				+ "where (NOT v.deleted = true)", DeveloperEntity.class).getResultList();
 		return result;
 	}
+	
+	private List<DeveloperEntity> getAllEntitiesIncludingDeleted() {
+		List<DeveloperEntity> result = entityManager.createQuery( "SELECT v from "
+				+ "DeveloperEntity v "
+				+ "LEFT OUTER JOIN FETCH v.address "
+				+ "LEFT OUTER JOIN FETCH v.contact "
+				+ "LEFT OUTER JOIN FETCH v.developerCertificationStatuses ", DeveloperEntity.class).getResultList();

```

Comment:
Not sure why we're joining in the developerCertificationStatuses here. They're not used in the report that this code is used for